.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   ignorePatterns: ["node_modules/"],
   rules: {
     "react/react-in-jsx-scope": "off",
+    'react/prop-types': ['off'],
```

Comment:
Bit of a nitpick, but could you remove the [] thanks!

Reply:
This line is here to fix this : **error  'href' is missing in props validation  react/prop-types**

Reply:
Sorry, I wasn't very clear here, but this change is unimportant, so I will just close it.